marshmallow_jsonapi/fields.py

```diff
@@ -22,6 +25,12 @@ class BaseRelationship(Field):
     pass
 
 
+def stringify(value):
```

Comment:
According to the spec:

>  The values of the id and type members MUST be strings.

http://jsonapi.org/format/#document-resource-object-identification

We *should* be stringifying IDs in the main data object as well. =/